docs/contributors/compiler.md

```diff
@@ -52,6 +52,9 @@ Usage: sql-to-dbsp [options] Input file to compile
     --ignoreOrder
       Ignore ORDER BY clauses at the end
       Default: false
+    --jdbcSource
```

Comment:
Confluent Schema Registry is available via HTTP not JDBC.
https://docs.confluent.io/platform/current/schema-registry/develop/using.html

Reply:
This PR doesn't handle the schema registry, that will be an upcoming PR.
I don't understand yet how the schema registry works, it doesn't behave like a normal database catalog.